x-pack/filebeat/module/gcp/audit/config/pipeline.js

```diff
@@ -85,6 +85,29 @@ function Audit(keep_original_message) {
         }
     };
 
+    // adding a boolean attribute for dry-run denied events in binary authorization.
+    // https://cloud.google.com/binary-authorization/docs/viewing-audit-logs#dry_run_events
+    // adding a boolean attribute for breakglass events in binary authorization
+    // https://cloud.google.com/binary-authorization/docs/viewing-audit-logs#view_breakglass_events
+    var checkBinaryAuthLabels = function(evt){
+        var labels = evt.Get("json.labels")
+
+        if(typeof labels === 'object' && labels !== null){
+           if ("imagepolicywebhook.image-policy.k8s.io/dry-run" in labels){
+                evt.Put("gcp.audit.binary_auth.dry_run_denied", true)
+           }
+           if("imagepolicywebhook.image-policy.k8s.io/break-glass" in labels){
+                evt.Put("gcp.audit.binary_auth.breakglass_used", true)
+           }
+           if("imagepolicywebhook.image-policy.k8s.io/overridden-verification-result" in labels){
+                var image = labels["imagepolicywebhook.image-policy.k8s.io/overridden-verification-result"]
+                image = image.substring(0,image.indexOf(':'))
```

Comment:
```suggestion
                image = image.substring(1,image.indexOf(':'))
```

After trying out like so: <img width="881" alt="image" src="https://github.com/unravelin/beats/assets/34165723/48b074d7-de87-4746-840b-a08800e624ed">

Reply:
Maybe `image.match(/'.*'/g)[0].slice(1, -1)` is even better to return the tag of the image too.